src/main/java/at/hannibal2/skyhanni/config/commands/Commands.kt

```diff
@@ -369,6 +370,10 @@ object Commands {
             "shtestburrow",
             "Sets a test burrow waypoint at your location"
         ) { GriffinBurrowHelper.setTestBurrow(it) }
+        registerCommand(
+            "shclearstoredplayers",
```

Comment:
I know finding a good name for this command is really hard, but the current name is confusing when not knowing its about island leave join messages. Too abstract/vague

Reply:
Do you have a suggestion?

Reply:
What if I improve the description?
"Clears the stored players on the island` for island leave/join messages`."

Reply:
maybe something like `/shclearislandplayers`?

Reply:
Another thing, is this only intended for testing purposes? The list already gets cleared when the player itself switches worlds, right?
Maybe we can remove the command entirely? That way we don't need to find hard names XD

Reply:
> Another thing, is this only intended for testing purposes? 

pretty much

Maybe I could make a command to trigger the lorenzworldchangeevent?